### PR TITLE
option parameter to skip modules updates

### PR DESCRIPTION
**Pull request by ayushin**
**Description of the change**

Allow skip modules updates with optional ODOO_SKIP_UPDATE="yes"

**Benefits**

It is not always practical, or even possible to update all the installed modules on each pod startup. We have a number of databases where the update would take over 30+ minutes so it is not feasible/possible to do it upon startup.

**Possible drawbacks**

None

### CODE REVIEW

File: 12/debian-10/rootfs/opt/bitnami/scripts/libodoo.sh [hunk 1]
```diff
@@ -165,8 +165,10 @@ odoo_initialize() {
             # So we need to clear the assets or if none of the .css/.js will load properly
             info "Clearing assets cache from the database"
             postgresql_remote_execute "${db_execute_args[@]}" <<< "DELETE FROM ir_attachment WHERE url LIKE '/web/content/%';"
-            info "Updating modules"
-            odoo_execute --update=all
+            if ! is_boolean_yes "$ODOO_SKIP_UPDATE"; then
```
Comment (pablogalegoc) [inline 12/debian-10/rootfs/opt/bitnami/scripts/libodoo.sh:168]: I would rename it to `ODOO_SKIP_MODULES_UPDATE` to make it clear it's not skipping the update of the application itself

**Review by pablogalegoc — COMMENTED** [review]
Hi @ayushin! Thanks for the suggestion! This seems a useful addition to the image, I'll leave some comments:

We should add this new variable to the list of available ones
https://github.com/bitnami/bitnami-docker-odoo/blob/7b8c9efeb6b5e76fb90ed22c27d6c55e43668e3c/12/debian-10/rootfs/opt/bitnami/scripts/odoo-env.sh#L25-L28

Set a default value of "no" like in
https://github.com/bitnami/bitnami-docker-odoo/blob/7b8c9efeb6b5e76fb90ed22c27d6c55e43668e3c/12/debian-10/rootfs/opt/bitnami/scripts/odoo-env.sh#L90-L91

And finally leave a brief description of it in the README
https://github.com/bitnami/bitnami-docker-odoo#user-and-site-configuration

Thank you!

**Comment by pablogalegoc** [comment]
Hi again @ayushin! Do you plan to look into this in the near future or should we take it?

**Comment by ayushin** [comment]
it would be great if you could finish it - bit short on time these days

**Comment by pablogalegoc** [comment]
We'll include this change in a coming release so I'll close this PR

**Pull request closed without merging**